packages/iris-grid/src/IrisGridUtils.test.ts

```diff
@@ -680,3 +667,131 @@ describe('dehydration methods', () => {
     ).toBe(true);
   });
 });
+
+describe('hydration methods', () => {
+  it.each([
+    [
+      'hydrateIrisGridPanelStateV1',
+      {
+        isSelectingPartition: false,
+        partition: null,
+        partitionColumn: 'INVALID',
+        advancedSettings: [],
+      },
+    ],
+    [
+      'hydrateIrisGridPanelStateV2',
+      {
+        isSelectingPartition: false,
+        partitions: [null],
+        partitionColumns: ['INVALID'],
+        advancedSettings: [],
+      },
+    ],
+  ])('%s invalid column error', (_label, panelState) => {
+    expect(() =>
+      IrisGridUtils.hydrateIrisGridPanelState(
+        irisGridTestUtils.makeModel(
+          irisGridTestUtils.makeTable({
+            columns: irisGridTestUtils.makeColumns(5, 'name_'),
+          })
+        ),
```

Comment:
This same model is used in all tests, you could declare it at the beginning of the describe.